api/templates/design.go

```diff
@@ -280,51 +282,60 @@ var _ = Service("templates", func() {
 			GET("/project/{branch}/instances/{instanceId}")
 			Meta("openapi:tag:instance")
 			Response(StatusOK)
+			InstanceNotFoundError()
 		})
 	})
 
 	Method("UpdateInstance", func() {
-		Meta("openapi:summary", "TODO")
+		Meta("openapi:summary", "Update instance name")
+		Result(InstanceDetail)
 		Payload(func() {
```

Comment:
Dodefinoval som endpointy ohladom instancii, ako potrebovalo UI.

![image](https://user-images.githubusercontent.com/19371734/165913859-1480a196-79b6-42b2-b749-23fdc4070749.png)


Je to nasadene: 
https://templates.keboola.com/v1/documentation/